src/control/lib/hardware/pci.go

```diff
@@ -514,8 +519,8 @@ func (b *PCIBus) IsZero() bool {
 
 func (d *PCIDevice) String() string {
 	var speedStr string
-	if d.LinkSpeed > 0 {
-		speedStr = fmt.Sprintf(" @ %.2f GB/s", d.LinkSpeed)
+	if d.LinkNegSpeed > 0 {
+		speedStr = fmt.Sprintf(" @ %.2f GT/s", d.LinkNegSpeed/1e+9)
```

Comment:
The divisor should probably use the same named const as the multiplier elsewhere. Or, probably better, define a `pciutils.LinkSpeedString()` function or something similar, as it will encapsulate responsibility for knowing these details in one place.

Reply:
used `humanize` library to decide the right text